style.css

```diff
@@ -0,0 +1,3 @@
+body {
+  font-family: Montserrat, sans-serif;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5568495)